castle/static/castle/jumbotron.css

```diff
@@ -39,6 +39,18 @@ div#news ul.meta li {
 	margin-right: 10px;
 }
 
+div.col-md-4 span.byline {
+	font-size: 12px;
+}
+
+span.byline {
+	font-style: italic;
```

Comment:
See the front page for an example. The italicised bylines cause a rapid alternation of "normal" formatted text and reduced-size italics that is hard to read and visually displeasing.